.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Build distribution
+
+on: [push, pull_request]
```

Comment:
should not this be on new release published?

Reply:
> should not this be on new release published?

On push and pull request It runs the build only the publishing happens, if there is a tag.